### DISCUSSION TITLE
(49): Can't assign requested address error how can i fix that error?

### DISCUSSION THREAD

**Discussion by Shoiion** [community/community#80295] [New to GitHub]
ERROR: Couldn't connect to 127.0.0.1:57955 - Please make sure you don't have an antivirus or firewall blocking this connection. (49): Can't assign requested address
error: unable to read askpass response from '/Applications/GitHub Desktop.app/Contents/Resources/app/desktop-trampoline/desktop-trampoline'
fatal: could not read Username for 'https://github.com': terminal prompts disabled

i am using GitHub Desktop on macOS, my port 57955 is free and there no process that use it,   i tried to reinstall GitHub desktop as well

**Comment by queenofcorgis** [comment]
Hello @Lav0chka 👋🏼 

Can you share what commands or any other details that may help someone assist?
